website/quickfiles/views.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+import requests
+from flask import send_from_directory, Response, stream_with_context
+
+from website.settings import EXTERNAL_EMBER_APPS, PROXY_EMBER_APPS
+
```

Comment:
Add routes list, e.g.
```
routes = [
    '/quickfiles/',
    '/<uid>/quickfiles/'
]
```
This way we can just append to this list as OSF things are moved into ember_osf_web